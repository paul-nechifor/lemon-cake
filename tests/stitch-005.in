(stitch :(5 2) :(88 123))
