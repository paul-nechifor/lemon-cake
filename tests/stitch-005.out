(5 2 88 123)
